src/hooks/useTotalPcs.ts

```diff
@@ -0,0 +1,36 @@
+import { BuyBy } from '@types';
+import { useAppSelector } from '@redux';
+import { ARRAY_INDEX_DIFF } from '@constants';
+
+const INITIAL_ZERO = 0;
```

Comment:
Better to have INITIAL_VALUE, INITIAL_AMOUNT, INITIAL_COUNT
What if the initial is 1 in the future ?